swagger.yaml

```diff
@@ -223,28 +266,54 @@ definitions:
       gap:
         description: "Mandatory gaps in trail (must be cleared)"
         type: "boolean"
-      
+
   Trail:
     type: "object"
     required:
     - "name"
     - "location"
```

Comment:
Is there any reason that "name" and "location" are required in Trail but not in Trails? Do you mean "zipCode" instead of "location"? I think you would need to put the "required" field in Features to specify that any of the fields in Features are required since Features is just a reference in Trails.